backend/test/.env

```
@@ -0,0 +1,7 @@
+NUM_PARALLEL_NODES=10
+TEST_LABEL="E2ECritical"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)